explorerlib/index.go

```diff
@@ -56,9 +61,14 @@ func (index *Index) BuildGoodQueries(wordCountsByLanguage map[string]WordCounts,
 			if len(response.Results) > 0 {
 				words = append(words, word)
 				i++
+				fmt.Printf("\rBuilding and validating queries: %.0f %% completed", (float32(i)/float32(numberOfQueryByLanguage))*100)
```

Comment:
This returns the progression for one language. It should give the progression for all languages. The progression should be (#languagesDone + i) / (#languages * numberOfQueryByLanguage) * 100